frontend/src/index.js

```diff
@@ -2,6 +2,7 @@ import ReactDOM from "react-dom";
 import { BrowserRouter } from "react-router-dom";
 
 import "./index.css";
+import "styles";
```

Comment:
This doesn't do much except breaking the build. Did your editor not show an error?

Reply:
You can also check the [logs](https://app.netlify.com/sites/peppy-peony-33d6b6/deploys/62e7f78973d2d10008fb83e2)

Reply:
<img width="869" alt="Screenshot 2022-08-02 at 07 41 09" src="https://user-images.githubusercontent.com/26620750/182300200-67a4b0f4-113d-4f82-8d3d-5262a606a7a4.png">

Reply:
What you want is to import these globals into the top level CSS file `index.css`


Which you can do so: `@import "styles/globals.css";`

Reply:
Got it